contribute/README.md

```diff
@@ -10,9 +10,13 @@ Thanks for enhancing OpenFisca anyway!
 
 ## Why contribute to OpenFisca?
 
-OpenFisca is a project being developed under the GPLv3 license or later.
+OpenFisca is a project being developed under the AGPL-3.0 license.
 The source code is freely available and modifiable.
 
+We ask for improvements and evolution of OpenFisca to be shared back, for it allows the community to grow through improvements to the _core engine_ and updates to the _modelled tax-benefit systems_ (country packages, extensions, reforms, etc.).
```

Comment:
Explain that this condition is a license effect?

Reply:
I believe it is rather the desired effect, and that the choice of license enforces it 🙂

Reply:
@MattiSG Yeah, of course. Would you further change wording to enforce that message ?